src/Processors/Formats/Impl/ArrowBlockInputFormat.h

```diff
@@ -3,6 +3,7 @@
 #if USE_ARROW
 
 #include <Processors/Formats/IInputFormat.h>
+#include <Processors/Formats/Impl/ArrowColumnToCHColumn.h>
```

Comment:
I think importing Impl dir in .h file is not good, but ArrowColumnToCHColumn (and CHColumnToArrowColumn) became statefull and I need to create their instances as class members. Maybe move these classes into another dir (I am not sure which one)?

Reply:
Maybe better to use `pimpl` here? And move this include to the .cpp file.

Reply:
That's a good idea, thanks